Artifacts/PgSql/Data/Security/2000-AdminApp-ClaimSets.sql

```diff
@@ -146,8 +157,10 @@ BEGIN
 
     RAISE NOTICE 'Updating educationStandards authorization strategy for READ.';
 
-    UPDATE dbo.resourceclaimauthorizationmetadatas
+    UPDATE dbo.ResourceClaimActionAuthorizationStrategies
     SET authorizationstrategy_authorizationstrategyid = authorization_strategy_id
+	INNER JOIN dbo.ResourceClaimActionAuthorizations
```

Comment:
update new table and columns here from this 
![image](https://user-images.githubusercontent.com/34613894/145424036-9f2d0ad6-1be6-4233-a053-721651e46440.png)